src/main/java/com/diffblue/deeptestutils/Reflector.java

```diff
@@ -378,9 +350,7 @@ private static void makePublic(final CtClass c) {
    * @param className a <code>String</code> giving the name of the class
    * @return an <code>Object</code> which is an instance of the specified class
    *
-   * @throws InvocationTargetException if the constructor of a class
-   * throws an exception
-   *
+   * @throws InvocationTargetException
```

Comment:
:no_entry_sign: What now throws an `InvocationTargetException`?

Reply:
While trying to address this comment I actually found another bug! https://diffblue.atlassian.net/browse/TG-5895
I updated the doc for this describing when we should throw an InvocationTargetException, and also added a TODO explaining what we're doing wrong and linking to the ticket.